quarkus-integration/quarkus/deployment/src/test/java/ai/timefold/solver/quarkus/TimefoldProcessorMultipleSolverPropertiesTest.java

```diff
@@ -0,0 +1,312 @@
+            .overrideConfigKey("quarkus.timefold.solver.\"solver1\".domain-access-type", "REFLECTION")
+            .overrideConfigKey("quarkus.timefold.solver.\"solver1\".termination.spent-limit", "4h")
+            .overrideConfigKey("quarkus.timefold.solver.\"solver1\".termination.unimproved-spent-limit", "5h")
+            .overrideConfigKey("quarkus.timefold.solver.\"solver1\".termination.best-score-limit", "0")
+            .overrideConfigKey("quarkus.timefold.solver.\"solver2\".daemon", "false")
+            .overrideConfigKey("quarkus.timefold.solver.\"solver2\".domain-access-type", "REFLECTION")
+            .overrideConfigKey("quarkus.timefold.solver.\"solver2\".termination.spent-limit", "1h")
+            .setArchiveProducer(() -> ShrinkWrap.create(JavaArchive.class)
+                    .addClasses(TestdataQuarkusEntity.class, TestdataQuarkusSolution.class,
+                            TestdataQuarkusConstraintProvider.class));
+
+    @Inject
+    @Named("solver1Config")
+    SolverConfig solverConfig;
+    @Inject
+    @Named("solver1Factory")
+    SolverFactory<TestdataQuarkusSolution> solver1Factory;
+    @Inject
+    @Named("solver1Manager")
+    SolverManager<TestdataQuarkusSolution, String> solver1Manager;
+
+    // ScoreManager per score type
+    @Inject
+    @Named("solver1ScoreManager")
+    ScoreManager<TestdataQuarkusSolution, SimpleScore> simpleScoreManager1;
+    @Inject
+    @Named("solver1ScoreManager")
+    ScoreManager<TestdataQuarkusSolution, SimpleLongScore> simpleLongScoreManager1;
+    @Inject
+    @Named("solver1ScoreManager")
+    ScoreManager<TestdataQuarkusSolution, SimpleBigDecimalScore> simpleBigDecimalScoreManager1;
+    @Inject
+    @Named("solver1ScoreManager")
+    ScoreManager<TestdataQuarkusSolution, HardSoftScore> hardSoftScoreManager1;
+    @Inject
+    @Named("solver1ScoreManager")
+    ScoreManager<TestdataQuarkusSolution, HardSoftLongScore> hardSoftLongScoreManager1;
+    @Inject
+    @Named("solver1ScoreManager")
+    ScoreManager<TestdataQuarkusSolution, HardSoftBigDecimalScore> hardSoftBigDecimalScoreManager1;
+    @Inject
+    @Named("solver1ScoreManager")
+    ScoreManager<TestdataQuarkusSolution, HardMediumSoftScore> hardMediumSoftScoreManager1;
+    @Inject
+    @Named("solver1ScoreManager")
+    ScoreManager<TestdataQuarkusSolution, HardMediumSoftLongScore> hardMediumSoftLongScoreManager1;
+    @Inject
+    @Named("solver1ScoreManager")
+    ScoreManager<TestdataQuarkusSolution, HardMediumSoftBigDecimalScore> hardMediumSoftBigDecimalScoreManager1;
+    @Inject
+    @Named("solver1ScoreManager")
+    ScoreManager<TestdataQuarkusSolution, BendableScore> bendableScoreManager1;
+    @Inject
+    @Named("solver1ScoreManager")
+    ScoreManager<TestdataQuarkusSolution, BendableLongScore> bendableLongScoreManager1;
+    @Inject
+    @Named("solver1ScoreManager")
+    ScoreManager<TestdataQuarkusSolution, BendableBigDecimalScore> bendableBigDecimalScoreManager1;
```

Comment:
`ScoreManager` is deprecated. We need not and should not support it in this new feature.
Also, aren't we missing the `SolverManager` here?

Reply:
Okay. I'll remove them. The `SolveManager` instances are injected at the lines 70 and 156.